index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+       <div class="logo"> <a><img src="images/image_geometry_1.png" alt="logo image"></a></div>
+        <nav>
+            <div class="Button">≡</div>
+        </nav>
+    </header>
+    <main>
+        <h1 class="intro">
+            Javed <br> Shahbaz
```

Comment:
- [OPTIONAL] I think it's better to use the font family provided by the Figma template. Will create a better UI. As not required by the previous reviewer I consider it optional.